installer/manifests/keptn/charts/control-plane/templates/api-gateway-nginx.yaml

```diff
@@ -370,6 +370,7 @@ spec:
       app.kubernetes.io/name: api-gateway-nginx
       app.kubernetes.io/instance: {{ .Release.Name }}
   replicas: 1
+  {{ include "control-plane.common.update-strategy" . }}
```

Comment:
missing indent function call?
this also goes for all other services where you made use of this helper function.